src/content/reference/react-dom/components/form.md

```diff
@@ -0,0 +1,445 @@
+---
+title: "<form>"
+canary: true
+---
+
+<Canary>
+
+React's `form` Component is currently only available in React's canary and experimental channels but you can use the [standard HTML `form` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) in your React's stable channel. Learn more about [React's release channels here](/community/versioning-policy#all-release-channels).
```

Comment:
This makes it sound like React provides a different form component, implying that you might need to import `form` from React. Maybe instead the wording should be something like "React's additions to the browser's `form` component are only in canary and experimental"

Reply:
Yeah, good note. I was thinking something similar. This copy definitely needs more nuance

Reply:
Fixed